utils/Azure/acrTools.ts

```diff
@@ -72,18 +72,30 @@ export async function getRepositoriesByRegistry(registry: Registry): Promise<Rep
     return allRepos;
 }
 
-/** Sends a custon html request to a registry
+/** List images under a specific digest */
+export async function getImagesByDigest(repo: Repository, digest: string): Promise<AzureImage[]> {
+    let allImages: AzureImage[] = [];
+    let image: AzureImage;
+    const { acrAccessToken } = await acquireACRAccessTokenFromRegistry(repo.registry, 'repository:' + repo.name + ':pull');
+    //const tags: TagInfo[] = await getTagAttributes('https://' + element.registry.loginServer, element.name, { bearer: acrAccessToken });
+    const tags: string[] = await getAssociatedTags('https://' + repo.registry.loginServer, repo.name, digest, { bearer: acrAccessToken });
+    for (let tag of tags) {
+        image = new AzureImage(repo, tag);
+        allImages.push(image);
```

Comment:
You may consider `allImages.push(new AzureImage(repo, tag))` if `image` is not used elsewhere.

Reply:
Yes, I did it like this to follow the other functions style in this file.
I'm fixing this in other functions as well.